package/MDAnalysis/analysis/encore/clustering/Cluster.py

```diff
@@ -0,0 +1,212 @@
+# Cluster.py --- classes to handle results of clustering runs
+# Copyright (C) 2014 Wouter Boomsma, Matteo Tiberti
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Ensemble representation --- :mod:`MDAnalysis.analysis.ensemble.ensemble`
+=====================================================================
+
+The module contains the Cluster and ClusterCollection classes which are
```

Comment:
How do we want to handle new clustering algorithms after encore was merged?  Wouldn't it be better to move the clustering algorithms and logic to a `analysis.cluster` module and have encore interact with that module. That would make it clearer for people to find who just want to do a clustering and make it obvious where new methods should be added.